plugins/source/crowdstrike/client/client.go

```diff
@@ -0,0 +1,72 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+type Client struct {
+	logger   zerolog.Logger
+	spec     specs.Source
+	Services Services
+}
+
+type Services struct {
+	Incidents Incidents
+	Alerts    Alerts
+}
+
+func (*Client) Logger() *zerolog.Logger {
+	return &log.Logger
+}
+
+func (*Client) ID() string {
+	return "CrowdStrikeClient"
```

Comment:
```suggestion
	return c.spec.Name
```
(`func (c *Client)` obvs...)